models/keep3r_network/keep3r_network_job_migration.sql

```diff
@@ -0,0 +1,66 @@
+{{ config (
+    tags=['dunesql']
+    , alias = alias('etv_job_migration')
+    , unique_key = ['blockchain', 'block_time']
```

Comment:
minor: unique key configs on all spells in this PR are not really needed, as they are views. most important for incremental spells.